src/scicookie/{{cookiecutter.project_slug}}/.makim.yaml

```
@@ -35,7 +35,7 @@ groups:
           jupyter-book build docs
           {%- elif cookiecutter.documentation_engine == "mkdocs" %}
           mkdocs build --config-file mkdocs.yaml
-          {%- elif cookiecutter.documentation_engine == "sphinx" %}
+          {%- elif cookiecutter.documentation_engine == "sphinx(rst) OR sphinx(rst)" %}
```

```suggestion
          {%- elif cookiecutter.documentation_engine.startswith("sphinx") %}
```

this condition `cookiecutter.documentation_engine == "sphinx(rst) OR sphinx(rst)"` will never be true .. because we don't have a documentation_engine with a name `sphinx(rst) OR sphinx(rst)`